dgbpy/mlapply.py

```diff
@@ -112,7 +112,7 @@ def computeScaler( infos, scalebyattrib, force=False ):
   return infos
 
 def getScaledTrainingData( filenm, flatten=False, scaler=dgbhdf5.Scaler.GlobalScaler, force=False, 
-                           nbchunks=1, split=None ):
+                           nbchunks=1, split=None, nbfolds=5 ):
```

Comment:
see the default to 1 here. Calling the function with all default arguments should return all the data in the example file